2_process.R

```diff
@@ -48,10 +49,31 @@ p2_targets_list <- list(
   # Pair PRMS segments with intersecting NHDPlusV2 reaches and contributing NHDPlusV2 catchments
   tar_target(
     p2_prms_nhdv2_xwalk,
-    create_GFv1_NHDv2_xwalk(prms_lines = p1_reaches_sf,nhd_lines = p1_nhdv2reaches_sf,prms_hrus = p1_catchments_sf,
+    create_GFv1_NHDv2_xwalk(prms_lines = p1_reaches_sf,nhd_lines = p1_nhdv2reaches_sf,
+                            prms_hrus = p1_catchments_sf_valid,
                             min_area_overlap = 0.5,drb_segs_spatial = drb_segs_spatial)
   ),
   
+  ## Melt PRMS_nhdv2_xwalk to get all cols of comids Ids and PRMS ids filtered to drb 
+  tar_target(p2_drb_comids_all_tribs, 
+            p2_prms_nhdv2_xwalk %>%
+              tidyr::separate_rows(comid_seg,sep=";") %>% 
+              rename(comid = comid_seg)
```

Comment:
For processing the land cover proportions (and any area-weighted variable from the MasterList), we'll want to use `comid_cat` here rather than `comid_seg` since `comid_seg` won't include all of the tributaries that contribute to the PRMS HRU. Here's a map for PRMS segid 112_1 that hopefully helps show the differences between the three columns in the xwalk table:
![example_segs](https://user-images.githubusercontent.com/8785034/151808239-4cb775eb-6f2d-4a77-b6d0-0eb5b13b3b3c.png)

Reply:
I'm realizing it's not clear from this map, but the segments shown in green are not represented by the PRMS lines (which overlaps the red/blue).

Reply:
that makes sense. Where's the PRMS seg 112_1 ?  Is that overlapping as well?

Reply:
PRMS seg 112_1 is just hidden under the red and blue segments in the map above.